app/services/request_key_manager.rb

```diff
@@ -5,4 +5,11 @@ class RequestKeyManager
       Figaro.env.saml_passphrase
     )
   end
+
+  cattr_accessor :equifax_ssh_key do
```

Comment:
why not indeed. I was just mirroring the method above, which does the exact same thing for a different key.

is the intent to move to constants instead of class variables everywhere in the code base? Maybe we should do that in a single PR since it reflects a syntax change, not a functionality change?